src/rt/sched/schedulerthread.h

```diff
@@ -267,33 +267,43 @@ namespace verona::rt
      */
     void mute_map_scan(bool force = false)
     {
-      for (auto entry = mute_map.begin(); entry != mute_map.end(); ++entry)
+      while (true)
```

Comment:
Why do you have to scan the mute map multiple times?  Could you add a comment to explain.I think it is because you might make something normal, which then requires removing from the mute map?

Reply:
Comment added. Yes, when a cown is unmuted it may have previously muted cowns that had scheduled behaviours on it. So their entries should be removed as well.